activiti-core/activiti-spring-boot-starter/src/main/java/org/activiti/spring/ApplicationDeployedEventProducer.java

```diff
@@ -76,10 +78,27 @@ private List<ApplicationDeployedEvent> getApplicationDeployedEvents() {
                         .latestVersion()
                         .list())
             .stream()
+            .map(this::withProjectVersion1Based)
             .map(deployment -> new ApplicationDeployedEventImpl(deployment, eventType))
             .collect(Collectors.toList());
     }
 
+    private Deployment withProjectVersion1Based(Deployment deployment) {
+        String projectReleaseVersion = deployment.getProjectReleaseVersion();
+        if(projectReleaseVersion != null && StringUtils.isNumeric(projectReleaseVersion)) {
```

Comment:
You can avoid this boilerplate code using only StringUtils.isNumeric(projectReleaseVersion), it already test for null: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNumeric-java.lang.CharSequence-